README.md

```diff
@@ -92,7 +92,7 @@ If you want to use any of the bundled icons, you need to add the icon fonts to y
 
 <br>
 
-_Note: you need to recompile your project after adding new fonts, also ensure that they also appear under **Copy Bundle Resources** in **Build Phases**._
+_Note: you need to recompile your project after adding new fonts, also ensure that they **NOT** appear under **Copy Bundle Resources** in **Build Phases**. (if necessary, remove the resources)_
```

Comment:
Why the `**NOT**`?

Reply:
When I installed the library and try to use occurred an error because the resources was in the Build Phase. To fixed the problem was necessary open the xcode, go to Build Phase and in the Copy Bundle Resources I deleted all resources with extension ".tff" (fonts type).

After deleted the resources and build the project again works prefectly.

Reply:
They need to be there though. Pretty sure they were re-added to that phase after re-adding them.

Reply:
I opened the xcode right now and the fonts resources is not in the Build Phase

![image](https://user-images.githubusercontent.com/8702612/86510413-428f3d00-bde7-11ea-8669-cad92e3b216d.png)

Reply:
I can try added the library in other project to check if I committed some mistake during the installation process, but in my current project is working without the fonts resources in the xcode... strange...

Reply:
Oh, yeah, you probably get them automatically from cocoapods. Back in the days people would link it manually instead.